src/prefabs/loading-object.js

```diff
@@ -1,16 +1,19 @@
 /** @jsx createElementEntity */
+import { BoxGeometry, Mesh, MeshBasicMaterial } from "three";
 import { createElementEntity } from "../utils/jsx-entity";
 import { loadModel } from "../components/gltf-model-plus";
 import loadingObjectSrc from "../assets/models/LoadingObject_Atom.glb";
-import { disposeNode } from "../utils/three-utils";
+import { cloneObject3D, disposeNode } from "../utils/three-utils";
 
 // TODO We should have an explicit "preload assets" step
-let loadingObject = new THREE.Mesh(new THREE.BoxGeometry(), new THREE.MeshBasicMaterial());
+let loadingObject = new Mesh(new BoxGeometry(), new MeshBasicMaterial());
 loadModel(loadingObjectSrc, null, true).then(gltf => {
   disposeNode(loadingObject);
   loadingObject = gltf.scene;
 });
 
+// TODO: Do we really need to clone the loadingObject every time?
```

Comment:
We may have more than one so we definitely need multiple. We could render it as instanced geometry which might be a good idea, probably even better than pooling.